running-automation-testing.md

```diff
@@ -0,0 +1,207 @@
+
+## 3. Setup Bamboo
+
+### Installation
+
+Download Bamboo from https://www.atlassian.com/software/bamboo. Follow their [installion guide](https://confluence.atlassian.com/bamboo/getting-started-with-bamboo-289277283.html?_ga=2.95849887.246880307.1531709232-1995250601.1528082340). 
+
+### Create a project
+
+1) Login to Bamboo. In the top navigation bar, hover over the 'Create' dropdown menu and select 'Create Project'. 
+
+![create-project](assets/create-project.png)
+
+2) Fill out the information about the project. 
+
+![config-proj](assets/config-proj.png)
+
+3) Create a new plan for your project. 
+
+![create-plan](assets/create-plan.png)
+
+4) Fill out the information about the plan. and click on 'Configure plan' when finished. 
+
+![configure-plan](assets/configure-plan.png)
+
+> Note: If you link to a GitHub repository and you use two-factor authentication, you must generate a personal access token in GitHub to use as the password
+
+### Create task to run automation test
+
+#### First task: npm install
+
+Now that we have created the plan, we must now configure a job. 
+
+![config-job](assets/config-job.png)
+
+Click on 'Add task'. 
+
+In this guideline, we will be running the NodeJS test script, so we need to select the task type 'npm'. 
+
+![npm-task](assets/npm-task.png)
+
+In the 'Node.js executable' section, make sure the executable is pointing to the directory that has your node version. You can do this by clicking 'Add a new executable'. 
+
+![new-executable](assets/new-executable.png)
+
+For the command, put 'install'. 
+
+Click 'Save'. 
+
+![npm-install](assets/npm-install.png)
+
+#### Second task: run automation script
+
+Add another task of type 'npm'. 
+
+In the 'Node.js executable' section, make sure the executable is the same as the 'npm install' task we created previously. 
+
+For the command, run your automation script. 
+
+If necessary, you can click on 'Advanced options' and provide environment variables. 
+
+![run-task](assets/run-task.png)
+
+Click 'Save'. 
+
+### Build
+
+Click on 'Create' to start your first build. 
+
+If your build is a failure, you can check the "Logs" tab. 
+
+![failure](assets/failure.png)
+
+To see further details on the log, click on 'Default Job'. 
+
+![default-job](assets/default-job.png)
+
+To edit your plan, you can click on it near the top of the screen, next to where it says 'Build dashboard'. 
+
+![go-to-job](assets/go-to-job.png)
+
+Then click on the edit pencil icon on the right side of the plan you wish to edit. 
+
+![edit-plan](assets/edit-plan.png)
+
+Now you can select any job you want to edit. 
+
+![select-job](assets/select-job.png)
+
+Once you are done editing, you can click on the 'Run' button near the top right corner of the screen. 
+
+![run-plan](assets/run-plan.png)
+
+If your build was successful, check Kobiton cloud devices to see if a test session was created. 
+
+## Get the automation session data through Kobiton REST API
```

Comment:
All of this is using terminal command right. What happened when I want to use other languages ?  Let's think about that and come to me when you have the idea

Reply:
https://api.kobiton.com/docs/      also includes the other languages other than terminal. I think either Bao or Duy said in my previous guidelines to use terminal as default

Reply:
So why don't we put a brief sentence in the top of the session to guide users ?
Suggestion: 
``` 
This session will use bash commands as the default language to demonstrate. Please visit [api docs]({PUT_API_DOCS_URL}) to....
```